docs/crash-reporting.md

```diff
@@ -12,6 +12,14 @@ Documentation for the specific libraries is included in the [Android Components
 
 The Glean crash ping format is documented [here](https://github.com/mozilla-mobile/android-components/blob/master/components/lib/crash/docs/metrics.md).
 
+## Breadcrumbs
+
+[Breadcrumbs](https://github.com/mozilla-mobile/android-components/blob/master/components/support/base/src/main/java/mozilla/components/support/base/crash/Breadcrumb.kt) are trail of events that are sent with each crash report to both Socorro and Sentry.  
+
+### Events
+
+In [HomeActivity](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/HomeActivity.kt) when `onDestinationChanged` occurs, the destination fragment's name and its custom tab value is added to the breadcrumbs. 
```

Comment:
"and whether it is a custom tab" (to make it clear it's a boolean)

Reply:
Thanks, I'll fix.